### PR TITLE
added new language support for the urdu 

### PR DESCRIPTION
**Pull request by talha7o**
added new language support for the urdu speaking community from India, Pakistan, Bangladesh and Afghanistan.

### CODE REVIEW

File: app/locales/ur/overlap.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "overlap_title": "اوورلیپ چیک کریں",
+  "overlap_para_1": "سبز حلقے آپ کے مقامات کا گچھا ہیں ، بڑا کرنے کے لئے یہاں دبائیں۔ لال نشان آپ کے اندراج شدہ مقامات ہیں۔ \ n \ n بخوبی جامنی رنگ کے حلقوں کے بطور دکھائے جانے والے تصدیق شدہ مقدمات کی عوامی معلومات ڈاؤن لوڈ کرنے کے لئے بٹن دبائیں۔",
```
Comment (alpita-masurkar) [inline app/locales/ur/overlap.json]: Two points,
1. there is a space between \ and n. There shouldn't be any space in the newline character (should be \n) 
2. The value of "overlap_para_1" was updated recently. Does it correctly translate to the following?
 "overlap_para_1": "The green trail represents your location history\n\nThe light purple circles represent the public dataset"

File: app/locales/ur/overlap.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "show_overlap": "پبلک ڈیٹا چیک کریں",
```
Comment (alpita-masurkar) [inline app/locales/ur/overlap.json]: for show overlap, is this the line translated: "show_overlap": "Click to view the public dataset",?
(This line had gotten updated recently)

**Comment by alpita-masurkar** [comment]
After the two fixes mentioned in the code comments are done, this pr is ready to be merged

**Comment by alpita-masurkar** [comment]
@talha7o Are you on the slack group? 
**We need to merge the files, I got it reviewed, but it's pending some fixes.**
Can you please fix the merge conflicts and also add the new updates that have happened since your last commit?
1. there is a space between \ and n. There shouldn't be any space in the newline character (should be \n)
The value of "**overlap_para_1**" was updated recently. Can you please update this?
"overlap_para_1": "The green trail represents your location history\n\nThe light purple circles represent the public dataset" is the new line.
2. this key:
"**show_overlap**": "Click to view the public dataset",
3. **Exportscreen.json** has new key-value pairs. Can you translate them to?

**Comment by talha7o** [comment]
No, i will join the slack group and push the updated commit asap

**Comment by alpita-masurkar** [comment]
Thank you @talha7o , if you can share your email id here, I'll send you an invite

**Comment by talha7o** [comment]
talha7j@gmail.com

**Comment by alpita-masurkar** [comment]
done

**Pull request merged**